src/AdminConsole/Services/Mail/DefaultMailService.cs

```diff
@@ -52,7 +52,7 @@ public async Task SendPasswordlessSignInAsync(string magicLink, string email)
         await _provider.SendAsync(message);
     }
```

Comment:
Github will only allow me to comment down here.

The method `SendPasswordlessSignInAsync` is unused now and can be deleted.

Reply:
You can shift+click to select a region and comment on a code block